mainapp/models.py

```diff
@@ -199,7 +199,7 @@ class Contributor(models.Model):
         verbose_name="District - ജില്ല"
     )
     name = models.CharField(max_length=100, verbose_name="Name - പേര്")
-    phone = models.CharField(max_length=10, verbose_name="Phone - ഫോണ്‍ നമ്പര്‍", validators=[RegexValidator(regex='^[6-9]\d{9}$', message='Please Enter 10 digit mobile number', code='invalid_mobile')])
+    phone = models.CharField(max_length=14, verbose_name="Phone - ഫോണ്‍ നമ്പര്‍", validators=[RegexValidator(regex='^((\+91|0)[\- ]{0,1})?[456789]\d{9}$', message='Please Enter 10 digit mobile number or landline as 0<std code><phone number>', code='invalid_mobile')])
```

Comment:
It's better to define the regex string as a constant and use it everywhere

Reply:
Good idea, it should be DRY, on it